sdcm/tester.py

```diff
@@ -1605,14 +1606,17 @@ def run_stress_thread(self, stress_cmd, duration=None, stress_num=1, keyspace_nu
     # pylint: disable=too-many-arguments
     def run_stress_cassandra_thread(
             self, stress_cmd, duration=None, stress_num=1, keyspace_num=1, profile=None, prefix='', round_robin=False,
-            stats_aggregate_cmds=True, keyspace_name=None, stop_test_on_failure=True, **_):
+            stats_aggregate_cmds=True, keyspace_name=None, stop_test_on_failure=True, loaders_tenant=0, **_):
```

Comment:
maybe we should pass around the `loader_set` and the `node_list`, instead of passing down the indexes ?

Reply:
`run_stress_cassandra_thread` function is called from cdc longevity directly.
I changed getting of tenant sets